packages/web3/src/transaction/sign-verify.test.ts

```diff
@@ -46,4 +46,117 @@ describe('transaction', function () {
     const signature = transactionSign(txHash, privateKey)
     expect(transactionVerifySignature(txHash, publicKey, signature)).toEqual(true)
   })
+
+  describe('transactionVerifySignature additional tests', () => {
+    const validTxHash = '8fc5f0d120b730f97f6cea5f02ae4a6ee7bf451d9261c623ea69d85e870201d2'
+    const validPubKey = '02625b26ae1c5f7986475009e4037b3e6fe6320fde3c3f3332bea11ecadc35dd13'
+    const validSignature = '78471e7c97e558c98ac307ef699ed535ece319102fc69ea416dbb44fbb3cbf9c42dbfbf4ce73eb68c5e0d66122eb25d2ebe1cf9e37ef4c4f4e7a2ed35de141bc'
+
+    it('should handle different key types', () => {
+      const ec = new EC.ec('secp256k1')
```

Comment:
We can use `PrivateKeyWallet.Random` to generate key pair.

Reply:
Ok I will implement this